modules/node-shared/src/main/scala/org/tessellation/node/shared/cli/snapshot.scala

```diff
@@ -44,7 +44,8 @@ object snapshot {
           timeTriggerInterval = 43.seconds,
           declarationTimeout = 50.seconds,
           declarationRangeLimit = 3L,
-          lockDuration = 10.seconds
+          lockDuration = 10.seconds,
+          EventCutterConfig(maxBinarySizeBytes = 20_000_000)
```

Comment:
@TheMMaciek My understanding is the value is arbitrary. @ryle-goehausen ?

Reply:
It's arbitrary at this point, shouldn't matter exact value so long as it's referenced from elsewhere. I think it might make sense to pull this out as a constant though, so it could be imported from SDK or otherwise used elsewhere, rather than as a config.

Reply:
@ryle-goehausen What do you mean by "imported from SDK"? It's indeed pulled out at the edge of the application, so it can be provided per use case now.

Reply:
Yeah, we can make it a constant so that it's not configurable actually (every node needs the same value here). I suggested the value `20971520` because in the implementation we are comparing count of bytes in the array, so effectively this number would be equivalent to 20MB of data.

Reply:
> @ryle-goehausen What do you mean by "imported from SDK"? It's indeed pulled out at the edge of the application, so it can be provided per use case now.

Available for use in the CL-0 code for submission validation to ensure an invalid snapshot is not submitted, that's all I mean.

Reply:
I changed the value to `20971520` -- it will show up when I push changes with state channel event parents test